demo/chaincode/fpc/Makefile

```diff
@@ -20,3 +20,6 @@ build: $(BUILD_DIR)
 
 clean:
 	-rm -rf $(BUILD_DIR)
+
+docker-build: clean
+	$(DOCKER) run -v ${PWD}:/project/src/github.com/hyperledger-labs/fabric-private-chaincode/demo/chaincode/fpc -w /project/src/github.com/hyperledger-labs/fabric-private-chaincode/demo/chaincode/fpc hyperledger/fabric-private-chaincode-cc-builder sh -c make build
```

Comment:
Without running the prerequisites, this fails. Could you make it fail gracefully providing some info (e.g., "need to build cc-builder container, check out the README")?